tools/release.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import re
+import sys
+
+from utils import (
+    run,
+    repository_dirty,
+    remote_exists,
+    matching_remote_branches,
+    local_tag_exists,
+    remote_tag_exists,
+    local_branch_matches_remote_branch,
+    configure_git_user,
+    create_local_tag,
+    push_tag,
+)
+
+log = logging.getLogger(__name__)
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%SZ",
+)
+
+SEMVER_MAJOR_MINOR_PATTERN = "(0|[1-9]\d*)\.(0|[1-9]\d*)"
```

Comment:
Please use leading underscore for constants which are not meant to be used outside of the module.

Reply:
I'll leave it as-is since I don't want to explicitly enforce that this is something internal at this point (and given [this previous comment](https://github.com/mesosphere/kudo-cassandra-operator/pull/26#discussion_r351832646) too).